golem-cli/src/model/text/worker.rs

```diff
@@ -776,6 +776,17 @@ impl TextView for PublicOplogEntry {
                     }
                 ));
             }
+            PublicOplogEntry::ChangePersistenceLevel(params) => {
```

Comment:
i think this will fail until we update to 1.2.2-dev.10

Reply:
ok, I thought it was part of your release yesterday